common/djangoapps/student/roles.py

```diff
@@ -246,6 +246,21 @@ def __repr__(self):
         return '<{}>'.format(self.__class__.__name__)
 
 
+@register_access_role
+class GlobalCourseCreatorRole(OrgRole):
+    """
+    A global course creator with access to all site's courses.
+    """
+    ROLE = 'global_course_creator'
+
+    def __init__(self, *args, **kwargs):
+        """
+        Args:
```

Comment:
Fix docstrings https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings

Reply:
I personally use such type of docstrings only if that's the pattern. All docstrings in this file are done this way so I do this to maintain consistency. If you want, I can fix it.

Reply:
Please fix it. We need to write standard code even if we making changes in an old legacy code.